### PR TITLE
Modify buttons aesthetics and ux

### PR DESCRIPTION
**Pull request by gsn16**

### CODE REVIEW

**Comment by gsn16** [comment]
Check https://codepen.io/gsn16/pen/rvNyoW

**Comment by gsn16** [comment]
Check https://codepen.io/gsn16/pen/rvNyoW

**Comment by gsn16** [comment]
Check https://codepen.io/gsn16/pen/rvNyoW

**Comment by kylelogue** [comment]
I'm not certain that this style addition would be a positive change for the framework.  Can you provide more information as to what you're trying to achieve?

File: src/scss/elements/buttons.scss [hunk 1]
```diff
@@ -49,9 +49,9 @@ input[type='submit'] {
     }
 
     &:hover {
-        opacity: 0.85;
+        opacity: 0.9;
         cursor: pointer;
-        transform: scale(1.025);
+        box-shadow: inset 0 -1px 1px $color-gray-400;
```
Comment (itsag) [inline src/scss/elements/buttons.scss]: I don't think box shadow is a great effect for hover on a button. It'll make the buttons look totally different if the theme is Flat

**Pull request closed without merging**